sdk/spring/spring-messaging-azure-servicebus/src/main/java/com/azure/spring/messaging/servicebus/core/DefaultServiceBusNamespaceProcessorFactory.java

```diff
@@ -172,7 +172,6 @@ private ServiceBusProcessorClient doCreateProcessor(String name,
             }
 
             ServiceBusProcessorClient client;
-            //TODO(yiliu6): whether to use shared ServiceBusClientBuilder
```

Comment:
we should not simply remove the TODOs if we haven't done them

Reply:
Let's track these issues in Github issues. I think that is better than the way we do here which looks annoying to raise the issues by SonarQube.